app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ActiveRecord::Base
+  enum locale: {
+      no_pref: 0,
+      en_US: 1,
+      hi_IN: 2
+  }
```

Comment:
reference to the previous comment

Reply:
Need to discuss about updating migration files as well.